<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IP testovač</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>800</width>
          <height>600</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="stackedWidgetPage1">
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>365</x>
            <y>80</y>
            <width>70</width>
            <height>181</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>icon.png</pixmap>
          </property>
         </widget>
         <widget class="QPushButton" name="internbutton">
          <property name="geometry">
           <rect>
            <x>410</x>
            <y>370</y>
            <width>200</width>
            <height>70</height>
           </rect>
          </property>
          <property name="text">
           <string>Interní blacklisty</string>
          </property>
         </widget>
         <widget class="QPushButton" name="blackbutton">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>370</y>
            <width>200</width>
            <height>70</height>
           </rect>
          </property>
          <property name="text">
           <string>Blacklisty IP</string>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>200</x>
            <y>290</y>
            <width>400</width>
            <height>50</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
font-size: 40px;
font-weight: 800;
}</string>
          </property>
          <property name="text">
           <string>BlackList</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page">
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>110</y>
            <width>500</width>
            <height>50</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
font-size: 40px;
font-weight: 800;
}</string>
          </property>
          <property name="text">
           <string>Hledání v blacklistech</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QLineEdit" name="input">
          <property name="geometry">
           <rect>
            <x>250</x>
            <y>190</y>
            <width>300</width>
            <height>40</height>
           </rect>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit {
background-color: #f8f9fa;
}</string>
          </property>
          <property name="placeholderText">
           <string>Zadejte IP adresu/rozsah</string>
          </property>
         </widget>
         <widget class="QPushButton" name="blackfinder">
          <property name="geometry">
           <rect>
            <x>350</x>
            <y>250</y>
            <width>100</width>
            <height>30</height>
           </rect>
          </property>
          <property name="text">
           <string>Hledat</string>
          </property>
         </widget>
         <widget class="QTextEdit" name="displayField">
          <property name="geometry">
           <rect>
            <x>200</x>
            <y>320</y>
            <width>400</width>
            <height>151</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QTextEdit {
background-color: #f8f9fa;
}</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page_2">
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>30</y>
            <width>500</width>
            <height>50</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
font-size: 40px;
font-weight: 800;
}</string>
          </property>
          <property name="text">
           <string>Interní blacklisty</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QLineEdit" name="input_2">
          <property name="geometry">
           <rect>
            <x>250</x>
            <y>100</y>
            <width>300</width>
            <height>40</height>
           </rect>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit {
background-color: #f8f9fa;
}</string>
          </property>
          <property name="placeholderText">
           <string>Zadejte IP adresu/rozsah</string>
          </property>
         </widget>
         <widget class="QPushButton" name="internfinder">
          <property name="geometry">
           <rect>
            <x>350</x>
            <y>160</y>
            <width>100</width>
            <height>30</height>
           </rect>
          </property>
          <property name="text">
           <string>Hledat</string>
          </property>
         </widget>
         <widget class="QTextEdit" name="display10">
          <property name="geometry">
           <rect>
            <x>210</x>
            <y>360</y>
            <width>180</width>
            <height>70</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QTextEdit {
background-color: #f8f9fa;
}</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QTextEdit" name="display11">
          <property name="geometry">
           <rect>
            <x>410</x>
            <y>360</y>
            <width>180</width>
            <height>70</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QTextEdit {
background-color: #f8f9fa;
}</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QTextEdit" name="display16">
          <property name="geometry">
           <rect>
            <x>610</x>
            <y>440</y>
            <width>180</width>
            <height>70</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QTextEdit {
background-color: #f8f9fa;
}</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QTextEdit" name="display8">
          <property name="geometry">
           <rect>
            <x>610</x>
            <y>280</y>
            <width>180</width>
            <height>70</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QTextEdit {
background-color: #f8f9fa;
}</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QTextEdit" name="display6">
          <property name="geometry">
           <rect>
            <x>210</x>
            <y>280</y>
            <width>180</width>
            <height>70</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QTextEdit {
background-color: #f8f9fa;
}</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QTextEdit" name="display14">
          <property name="geometry">
           <rect>
            <x>210</x>
            <y>440</y>
            <width>180</width>
            <height>70</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QTextEdit {
background-color: #f8f9fa;
}</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QTextEdit" name="display7">
          <property name="geometry">
           <rect>
            <x>410</x>
            <y>280</y>
            <width>180</width>
            <height>70</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QTextEdit {
background-color: #f8f9fa;
}</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QTextEdit" name="display17">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>520</y>
            <width>180</width>
            <height>70</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QTextEdit {
background-color: #f8f9fa;
}</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QTextEdit" name="display5">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>280</y>
            <width>180</width>
            <height>70</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QTextEdit {
background-color: #f8f9fa;
}</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QTextEdit" name="display3">
          <property name="geometry">
           <rect>
            <x>410</x>
            <y>200</y>
            <width>180</width>
            <height>70</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QTextEdit {
background-color: #f8f9fa;
}</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QTextEdit" name="display4">
          <property name="geometry">
           <rect>
            <x>610</x>
            <y>200</y>
            <width>180</width>
            <height>70</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QTextEdit {
background-color: #f8f9fa;
}</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QTextEdit" name="display13">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>440</y>
            <width>180</width>
            <height>70</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QTextEdit {
background-color: #f8f9fa;
}</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QTextEdit" name="display15">
          <property name="geometry">
           <rect>
            <x>410</x>
            <y>440</y>
            <width>180</width>
            <height>70</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QTextEdit {
background-color: #f8f9fa;
}</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QTextEdit" name="display1">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>200</y>
            <width>180</width>
            <height>70</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QTextEdit {
background-color: #f8f9fa;
}</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QTextEdit" name="display12">
          <property name="geometry">
           <rect>
            <x>610</x>
            <y>360</y>
            <width>180</width>
            <height>70</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QTextEdit {
background-color: #f8f9fa;
}</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QTextEdit" name="display2">
          <property name="geometry">
           <rect>
            <x>210</x>
            <y>200</y>
            <width>180</width>
            <height>70</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QTextEdit {
background-color: #f8f9fa;
}</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QTextEdit" name="display9">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>360</y>
            <width>180</width>
            <height>70</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QTextEdit {
background-color: #f8f9fa;
}</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QTextEdit" name="displayStatus">
          <property name="geometry">
           <rect>
            <x>410</x>
            <y>540</y>
            <width>381</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QTextEdit {
background-color: #f8f9fa;
}</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>820</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
